src/main/java/htsjdk/samtools/SAMTagUtil.java

```diff
@@ -27,10 +27,12 @@
 
 /**
  * Facility for converting between String and short representation of a SAM tag.  short representation
- * is used by SAM JDK internally and is much more efficient.  Callers are encouraged to obtain the short
+ * is used by Htsjdk internally and is much more efficient.  Callers are encouraged to obtain the short
  * value for a tag of interest once, and then use the SAMRecord attribute API that takes shorts rather than
  * Strings.
  *
+ * Tags that are defined by the SAM spec are included in the enum {@link SAMTag} along with their precomputed short tag.
+ *
  * @author alecw@broadinstitute.org
  */
 public final class SAMTagUtil {
```

Comment:
You should perhaps consider deprecating this entire class, and moving the 2 non-deprecated methods to a different location.

Reply:
done